Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -56,7 +54,7 @@ CustomerSatisfactionPresenter >> initializePresenters
 
 
 `SpPresenter` defines messages for the creation of standard presenters: `newButton`, `newCheckBox`, `newDropList`, ...
-All of these are defined in the `widgets` protocol. These are shortcuts to create presenter for the associated widgets.
+All of these are defined in the `scripting - widgets` protocol of the `#SpTPresenterBuilder` trait. These are shortcuts to create presenters for the corresponding widgets.
```

Comment:
I checked this in a Pharo 12 image.

Reply:
I would suggest to remove all the protocol references because this is not useful and may change, else the rest is good.